src/main/java/io/reactivex/Flowable.java

```diff
@@ -8189,6 +8189,13 @@ public final Completable concatMapCompletableDelayError(Function<? super T, ? ex
      * source Publisher that are followed by another item within a computed debounce duration.
      * <p>
      * <img width="640" height="425" src="https://raw.github.com/wiki/ReactiveX/RxJava/images/rx-operators/debounce.f.png" alt="">
+     * <p>
+     * Delivery of the item after the grace period happens on the default {@code Scheduler}'s
```

Comment:
Selector-based debounces don't have explicit threading so the wording should be adjusted:

```
The delivery of the item happens on the thread of the first {@code onNext} or {@code onComplete} signal of the generated {@code Publisher}|{@code ObservableSource} sequence,
which if takes too long, a newer item may arrive from the upstream, causing the
generated sequence to get cancelled|disposed, [...]
```